### PR TITLE
Added automatic configuration of openATTIC

### PR DESCRIPTION
**Pull request by rjfd**
This PRs adds to the openattic installation the automatic configuration of the salt-api credentials and the configuration of the grafana hostname.

### CODE REVIEW

**Comment by LenzGr** [comment]
LGTM, thanks!

File: srv/modules/runners/sharedsecret.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import os
+
+
+def show():
+    filename = '/etc/salt/master.d/sharedsecret.conf'
+    if not os.path.exists(filename):
+        return None
+    with open(filename, 'r') as fhandler:
+        line = fhandler.readline()
+        idx = line.find(':')
```
Comment (votdev) [inline srv/modules/runners/sharedsecret.py]: Maybe you should add a comment why it is necessary to parse and split the line. I don't know how the file  looks like, so an example would be helpful to understand the code.

File: srv/modules/runners/sharedsecret.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+import os
+
+
+def show():
+    filename = '/etc/salt/master.d/sharedsecret.conf'
```
Comment (votdev) [inline srv/modules/runners/sharedsecret.py]: I would like to suggest to name the file salt_api_secret.conf to make it clear for what this file is used.

Comment (rjfd) [inline srv/modules/runners/sharedsecret.py]: That I cannot do. This is an "internal" file expected by the sharedsecret eauth module.

File: srv/modules/runners/sharedsecret.py (hunk 1 above)
Comment (rjfd) [inline srv/modules/runners/sharedsecret.py]: Yes, that would great. Thanks for the suggestion.

File: srv/modules/runners/sharedsecret.py (hunk 2 above)
Comment (votdev) [inline srv/modules/runners/sharedsecret.py]: OK, i thought it is a user generated file.

**Pull request merged**